docker-compose/config/worker-ubuntu16-04.jsonnet

```diff
@@ -2,35 +2,36 @@ local common = import 'common.libsonnet';
 
 {
   blobstore: common.blobstore,
-  browserUrl: common.browserUrl,
   maximumMessageSizeBytes: common.maximumMessageSizeBytes,
   scheduler: { address: 'scheduler:8983' },
-  httpListenAddress: ':7986',
+  httpListenAddress: common.httpListenAddress,
```

Comment:
Is this change intentional? I think the Docker Compose based deployments don't try to renumber TCP ports. You also haven't updated docker-compose.yml to use the new port number?

Reply:
It wasn't intentional. Reverting.